sci-geosciences/grass/ChangeLog

```diff
@@ -0,0 +1,615 @@
+# ChangeLog for sci-geosciences/grass
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  28 Mar 2015; Ted Tanberry <ted.tanberry@gmail.com> +grass-7.0.0.ebuild:
+  Version bump
```

Comment:
Are you using echangelog for this entry? For new versions there is always "*grass-7.0.0 (28 March 2015)"